ballerina/annotation_processor.bal

```diff
@@ -14,6 +14,13 @@
 // specific language governing permissions and limitations
 // under the License.
 
+isolated function getCorsConfig(GraphqlServiceConfig? serviceConfig) returns CorsConfig {
+    if serviceConfig is GraphqlServiceConfig && serviceConfig.cors is CorsConfig {
+        return <CorsConfig> serviceConfig.cors;
+    }
+    return {};
```

Comment:
We can return `CorsConfig?` here?

Reply:
As per the api docs, the default value of HTTP cors config is `{}`. Shall we keep this as it is?